### PR TITLE
Update scripts to be more easily usable

### PR DESCRIPTION
**Pull request by ElliotFriedman**

### CODE REVIEW

File: smart-contracts/scripts/bulkSetTokenLockBurnLimit.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ module.exports = async (cb) => {
     try {
         const accounts = await web3.eth.getAccounts();
 
-        bridgeBank = await BridgeBank.deployed()
+        bridgeBank = await BridgeBank.at(process.env.ADDRESS)
```
Comment (banshee) [inline smart-contracts/scripts/bulkSetTokenLockBurnLimit.js:66]: Maybe use a more specific environment variable - BRIDGEBANK_ADDRESS?

File: smart-contracts/scripts/bulkSetTokenLockBurnLimit.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ module.exports = async (cb) => {
...
         console.log(await bridgeBank.bulkWhitelistUpdateLimits(addresses, limits, {
             from: accounts[0],
             gas: 300000 // 300,000 gas
```
Comment (banshee) [inline smart-contracts/scripts/bulkSetTokenLockBurnLimit.js:69]: Do we need a higher limit on ropsten or mainnet?

**Review by banshee — DISMISSED** [review]
Comments are just minor points, wouldn't block merge

File: smart-contracts/scripts/bulkSetTokenLockBurnLimit.js (hunk 2 above)
Comment (ElliotFriedman) [reply smart-contracts/scripts/bulkSetTokenLockBurnLimit.js:69]: We can bump that up if needed when sending huge whitelist requests.

**Review by banshee — APPROVED** [review]

**Pull request closed without merging**